### ISSUE TITLE
Set kMDItemVersion on mac os app

### ISSUE THREAD

**Issue by laduke** [zerotier/ZeroTierOne#2340]
MDMs like Jamf read the kMDItemVersion information for reporting and patch management. But because all clients have the same version, it can’t find clients with outdated versions.

```
% zerotier-cli info
200 info AABBCCDDEE 1.14.0 ONLINE

mdls -name kMDItemVersion /Applications/ZeroTier.app 
kMDItemVersion = "1.0"

mdls -name kMDItemVersion /Library/Application\ Support/ZeroTier/One/zerotier-one                                                                        ~/repos/github.com/zerotier/Central
kMDItemVersion = (null)
```

I don't know yet where in xcode this is set. We already set the version to the right number in various places in xcode.

**Comment by glimberg** [comment]
In the desktop app, it's in the `Info.plist`

**Comment by adamierymenko** [comment]
It's actually the key `CFBundleShortVersionString` in Info.plist, will be easy to update on build.

**Comment by adamierymenko** [comment]
Closing since this is fixed in 1.14.1 release.
